packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -192,38 +178,64 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
     );
   };
 
+  private _selectedKey(): string | undefined {
+    if (this._isKeyValid(this.props.selectedKey)) {
```

Comment:
Still need to test this, but I suspect this new implementation will completely break the ability to use Pivot as "uncontrolled" since it always prefers the selected key from props rather than state.